tests/Nox.Types.Tests/EntityFrameworkTests/TestWithSqlite.cs

```diff
@@ -6,23 +6,39 @@ namespace Nox.Types.Tests.EntityFrameworkTests;
 public abstract class TestWithSqlite : IDisposable
 {
     private const string InMemoryConnectionString = "DataSource=:memory:";
+    //private const string InMemoryConnectionString = @"DataSource=test.db";
     private readonly SqliteConnection _connection;
 
-    protected readonly SampleDbContext DbContext;
+    protected SampleDbContext DbContext;
 
     protected TestWithSqlite()
     {
+        var c = Environment.CurrentDirectory;
```

Comment:
why this c?

Reply:
will remove, just to see where the database file was being saved